lib/std/mem.zig

```diff
@@ -1439,9 +1439,13 @@ pub fn writeInt(comptime T: type, buffer: *[@divExact(@typeInfo(T).Int.bits, 8)]
 pub fn writeIntSliceLittle(comptime T: type, buffer: []u8, value: T) void {
     assert(buffer.len >= @divExact(@typeInfo(T).Int.bits, 8));
 
-    if (@typeInfo(T).Int.bits == 0)
+    if (@typeInfo(T).Int.bits == 0) {
         return set(u8, buffer, 0);
-
+    } else if (@typeInfo(T).Int.bits == 8) {
```

Comment:
Should this be `<= 8`?

Reply:
I don't think so because, like said in the doc comment
```/// The bit count of T must be divisible by 8.```
and the assertion with the @divExact should fail on the bit count of the int being not divisible by 8
```assert(buffer.len >= @divExact(@typeInfo(T).Int.bits, 8));```